model/modelpb/event.pb.json_test.go

```diff
@@ -64,14 +74,16 @@ func TestEventToModelJSON(t *testing.T) {
 				},
 				Duration: int64(3 * time.Second),
 				Severity: 4,
+				Received: modeljson.Time(now),
 			},
 		},
 	}
 	for name, tc := range testCases {
 		t.Run(name, func(t *testing.T) {
 			var out modeljson.Event
 			tc.proto.toModelJSON(&out)
-			diff := cmp.Diff(*tc.expected, out)
+			diff := cmp.Diff(*tc.expected, out,
+				cmpopts.IgnoreUnexported(modeljson.Time{}))
```

Comment:
**issue**: since we're adding the `received` fields in the above structs, I think we should compare them in the the diff. Could you add a custom comparer ? Something like this should work:
```
cmp.Comparer(func(a modeljson.Time, b modeljson.Time) bool {
	return time.Time(a).Compare(time.Time(b)) == 0
}),
```